engine/src/main/java/org/entando/entando/aps/system/init/DatabaseManager.java

```diff
@@ -391,6 +391,59 @@ public void initComponentDefaultResources(Component componentConfiguration,
         }
     }
 
+    public void uninstallComponentResources(Component componentConfiguration,
+                                            SystemInstallationReport report) throws ApsSystemException {
+        String logPrefix = "|   ";
```

Comment:
A general improvement on this class: we could move this string in a static field, since it is used in various method.